cfme/containers/provider/__init__.py

```diff
@@ -440,24 +439,23 @@ def step(self):
 
 class ContainerObjectAllBaseView(ProvidersView):
     """Base class for container object All view.
-    TITLE_TEXT should be defined in child."""
-
+    summary should be defined in child as following:
+        * Text('//h1[normalize-space(.) = {TITLE_TEXT}]')
+    """
     policy = Dropdown('Policy')
     download = Dropdown('Download')
 
     @property
     def table(self):
         return self.entities.elements
 
+    @property
     def title(self):
-        if not hasattr(self, 'TITLE_TEXT'):
-            raise Exception('TITLE_TEXT is not defined for destination {} ("All").'
-                            .format(self.__class__.__name__))
-        return Text('//h1[normalize-space(.) = {}]'.format(quote(self.TITLE_TEXT)))
+        return self.summary
 
     @property
     def is_displayed(self):
-        return self.title.is_displayed
+        return self.summary.is_displayed
```

Comment:
@gshefer, I think you need to define one widget summary = Text('//h1[blabla]') and create is_displayed for each view. 
Where you can compare summary.text == 'Some View Title'